RELEASES.md

```diff
@@ -22,172 +26,230 @@ Below are the versioning schemes used for publishing debs:
- 12. Once accepted into `-proposed` by an SRU vanguard [ubuntu-advantage-tools shows up in the pending_sru page](https://people.canonical.com/~ubuntu-archive/pending-sru.html), check `rmadison ubuntu-advantage-tools | grep -proposed` to see if the upload exists in -proposed yet.
- 13. Confirm availability in <devel>-proposed pocket via
-```bash
-cat > setup_proposed.sh <<EOF
-#/bin/bash
-mirror=http://archive.ubuntu.com/ubuntu
-echo deb \$mirror \$(lsb_release -sc)-proposed main | tee /etc/apt/sources.list.d/proposed.list
-apt-get update -q;
-apt-get install -qy ubuntu-advantage-tools;
-apt-cache policy ubuntu-advantage-tools;
-EOF
-
-lxc launch ubuntu-daily:impish dev-i;
-lxc file push setup_proposed.sh
-lxc exec dev-i -- bash /setup_proposed.sh
-```
- 14. Once [ubuntu-advantage-tools shows up in the pending_sru page](https://people.canonical.com/~ubuntu-archive/pending-sru.html), perform the [Ubuntu-advantage-client SRU verification steps](https://wiki.ubuntu.com/UbuntuAdvantageToolsUpdates). This typically involves running all behave-(vm|lxd|gcp|azure|upgrade) targets with UACLIENT_BEHAVE_ENABLE_PROPOSED=1.
- 15. When SRU verification is complete and logs attached to all SRU-related bugs, mark any SRU-related bugs with 'verification-done' tags as each bug, task, or release verified successfully.
- 16. Once all SRU bugs are tagged as `verification*-done`, all SRU-bugs should be listed as green in [the pending_sru page](https://people.canonical.com/~ubuntu-archive/pending-sru.html). It is now time to ping the [current SRU vanguard](https://wiki.ubuntu.com/StableReleaseUpdates#Publishing) for acceptance of ubuntu-advantage-tools into -updates
+## Process
 
-## Ubuntu PRO Release Process
 
-Below is the procedure used to release ubuntu-advantage-tools to Ubuntu PRO images:
+### Background
 
- 1. [Open Daily PPA copy-package operation](https://code.launchpad.net/~ua-client/+archive/ubuntu/daily/+copy-packages)
- 2. Check Trusty, Xenial, Bionic package
- 3. Select Destination PPA: UA Client Premium [~ua-client/ubuntu/staging]
- 4. Select Destination series: The same series
- 5. Copy options: "Copy existing binaries"
- 6. Click Copy packages
- 7. Notify Pro Image creators about expected Premium PPA version (patviafore/powersj)
+The release process for ubuntu-advantage-tools has three overarching steps/goals.
 
+1. Release to our team infrastructure. This includes Github and the `ua-client` PPAs.
+2. Release to the latest ubuntu devel release.
+3. Release to the supported ubuntu past releases via [SRU](https://wiki.ubuntu.com/StableReleaseUpdates) using the [ubuntu-advantage-tools specific SRU process](https://wiki.ubuntu.com/UbuntuAdvantageToolsUpdates).
```

Comment:
Maybe we can here that the user need to have dput properly configured in the system to allow running sending the packages into our ppas

Reply:
Good point I forgot to bring that in - I added it to the Prerequisites list.